Purchases/Public/RCPurchasesErrorUtils.m

```diff
@@ -121,6 +121,10 @@
             return @"INVALID_APPLE_SUBSCRIPTION_KEY";
         case RCIneligibleError:
             return @"INELIGIBLE_ERROR";
+        case RCInsufficientPermissionsError:
+            return @"INSUFFICIENT_PERMISSIONS_ERROR";
+        case RCPaymentPendingError:
+            return @"payment_pending_error";
```

Comment:
uppercase, just to be consistent

Reply:
you know, I was convinced that I'd done this in uppercase. fixing.